### PR TITLE
Friendlier PR template

### PR DESCRIPTION
**Pull request by rjkerrison**
# Nice friendly PR template

Motivated by #334, which moved us away from airtable and further into public githubness. Also relates to #378.

### Acceptance Criteria

- [ ] Remove references to Airtable
- [ ] Include reference to testing strategy
- [ ] More clarity for an open source contributor

### Testing information

Give it a read – is it an improvement for an open source contributor?

### Pre-flight Checklist

- [ ] Testing aligns with [testing strategy][testing strategy]
- [ ] DUXD review approval
- [ ] Accessibility Reviewed
- [ ] Product owner demo

### Pre-merge Checklist

- [ ] Branch test approved

[testing strategy]: https://github.com/MHRA/products/blob/master/docs/principles/testing.md "MHRA/products testing strategy"

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,24 +1,28 @@
-### Name of the story
+# A nice friendly title for what this PR does
 
-[Airtable ticket](https://airtable.com/####)
+Relates to issue #(an issue)/bug found due to #(previous pull request).
```
Comment (TimboTambo) [inline .github/pull_request_template.md]: Might be nice to start with a label for what this is, e.g. `Short description of what the PR covers:...`

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+- [ ] Testing aligns with [testing strategy][testing strategy]
 - [ ] DUXD review approval
 - [ ] Accessibility Reviewed
 - [ ] Product owner demo
 
 ### Pre-merge Checklist
 
 - [ ] Branch test approved
+
+[testing strategy]: https://github.com/MHRA/products/blob/master/docs/principles/testing.md "MHRA/products testing strategy"
```
Comment (TimboTambo) [inline .github/pull_request_template.md]: Is there a benefit to having this link here, rather than inline where it's mentioned above?

Comment (rjkerrison) [reply .github/pull_request_template.md]: It's a habit of mine in markdown for three reasons:

1. It makes the paragraphs more readable not to have to read past potentially lengthy URLs and their titletext
2. It allows for reusing URLs
3. It puts the part of the file for updating them into one spot, like declaring hardcoded `const` at the top of a `.js`

File: .github/pull_request_template.md (hunk 1 above)
Comment (rjkerrison) [reply .github/pull_request_template.md]: How is it now?

Comment (TimboTambo) [reply .github/pull_request_template.md]: Great.

**Pull request merged**